src/crypto/qat/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+##
+# QAT wrapper for Ceph
+##
+
+# Build QAT driver library first
+include(QatDrv.cmake)
+
+set(qat_crypto_plugin_srcs
+  qat_crypto_accel.cc
+  qat_crypto_plugin.cc
+  qcccrypto.cc)
+
+add_library(ceph_crypto_qat SHARED ${qat_crypto_plugin_srcs})
+add_dependencies(ceph_crypto_qat QatDrv)
+target_include_directories(ceph_crypto_qat PRIVATE
+  ${CMAKE_CURRENT_BINARY_DIR}/src/QatDrv/quickassist/include
+  ${CMAKE_CURRENT_BINARY_DIR}/src/QatDrv/quickassist/lookaside/access_layer/include
+  ${CMAKE_CURRENT_BINARY_DIR}/src/QatDrv/quickassist/include/lac
+  ${CMAKE_CURRENT_BINARY_DIR}/src/QatDrv/quickassist/utilities/libusdm_drv
+  ${CMAKE_CURRENT_BINARY_DIR}/src/QatDrv/quickassist/utilities/libusdm_drv/linux/include)
```

Comment:
better off exposing these include directories in `QatDrv.cmake` as `QatDrv_INCLUDE_DIRS`.